pom.xml

```diff
@@ -34,7 +34,7 @@
 
     <licenses>
         <license>
-            <name>Apache 2.0 License</name>
+            <name>Apache License 2.0</name>
```

Comment:
I'd use `Apache-2.0` to be consistent with the ASF parent POM.
However, I think the `<licenses>` tag should be removed completely, as it's already inherited from the ASF parent.  This must be a remaining artifact from before the move to the ASF.